src/hooks/v2v3/transactor/DeploySplitsPayerTx.ts

```diff
@@ -0,0 +1,68 @@
+import * as constants from '@ethersproject/constants'
+import { t } from '@lingui/macro'
+import { DEFAULT_MEMO } from 'constants/transactionDefaults'
+import { ProjectMetadataContext } from 'contexts/shared/ProjectMetadataContext'
+import { TransactionContext } from 'contexts/Transaction/TransactionContext'
+import { V2V3ContractsContext } from 'contexts/v2v3/Contracts/V2V3ContractsContext'
+import { TransactorInstance } from 'hooks/Transactor'
+import { useWallet } from 'hooks/Wallet'
+import { useContext } from 'react'
+import { useV2ProjectTitle } from '../ProjectTitle'
+
+export type DeploySplitsPayerTxArgs = {
+  customBeneficiaryAddress: string | undefined
+  customMemo: string | undefined
+  tokenMintingEnabled: boolean
+  preferClaimed: boolean
+}
+
+export function useDeploySplitsPayerTx(): TransactorInstance<DeploySplitsPayerTxArgs> {
+  const { transactor } = useContext(TransactionContext)
+  const { contracts } = useContext(V2V3ContractsContext)
+  const { userAddress } = useWallet()
+  const { projectId } = useContext(ProjectMetadataContext)
+
+  const projectTitle = useV2ProjectTitle()
+
+  const DEFAULT_METADATA = [0x1]
```

Comment:
Does this work instead

https://github.com/jbx-protocol/juice-interface/blob/58db3e06ee8438abfab88a22386e03a4368eb956/src/constants/transactionDefaults.ts#L2